finetune/target_models/sequence_labeling.py

```diff
@@ -193,6 +205,25 @@ def _initialize(self):
         self.multi_label = self.config.multi_label_sequences
         return super()._initialize()
 
+    def finetune(self, Xs, Y=None, context=None, update_hook=None):
+        if self.config.auto_negative_sampling and Y is not None:
+            model_copy = copy.deepcopy(self)
```

Comment:
Should we decrease CPU threshold for the scheduler accordingly because of this copy?

Reply:
This copy *should* be quite light, because basically only the config has been populated at this point.